tutorials/cyber/raw_data_generator/data_generator.yml

```diff
@@ -0,0 +1,45 @@
+name: data_generator
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - boost-cpp=1.66.0=1
+  - bzip2=1.0.6=h14c3975_1002
+  - ca-certificates=2019.3.9=hecc5488_0
+  - certifi=2019.3.9=py36_0
+  - curl=7.62.0=hbc83047_0
+  - gettext=0.19.8.1=hc5be6a0_1002
+  - hdfs3=0.3.0=py36_1000
```

Comment:
are these all needed for this notebook? Looks like you might have used them in your dev environment, but this notebook includes local copies of the data, right?

Reply:
Updated the file